index.html

```diff
@@ -25,16 +26,32 @@
         <h1>Scott Brenden</h1>
     </header>
     <nav>
```

Comment:
**TODO FOR LOST POINTS**

At some point as we move into your mobile-first view, your nav should become a hamburger menu per the instructions.